pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -426,6 +426,19 @@ default void configureSchemaInfo(String topic, String componentName,
         return DefaultImplementation.newAutoProduceSchema(schema);
     }
 
+    /**
+     * Create a schema instance that accepts a serialized Avro payload
+     * without validating it against the schema specified. 
+     * It can be useful when migrating data from existing event or message stores.
+     *
+     * @return the auto schema instance
+     * @since 2.9.0
+     * @see #AUTO_PRODUCE_BYTES()
+     */
+    static Schema<byte[]> AUTO_PRODUCE_VALIDATED_AVRO_BYTES(Schema<?> schema) {
```

Comment:
why not adding simply a method that creates a AUTO_PRODUCE_BYTES schema without validation ?

```
    static Schema<byte[]> AUTO_PRODUCE_BYTES(Schema<?> schema, boolean validate) {
        return DefaultImplementation.newAutoProduceSchema(schema, validate);
    }
```
this "feature" will be useful for other Schemas other than Avro

@jerrypeng @sijie @codelipenghui @congbobo184

Reply:
my idea was exactly to prevent other schema formats from abusing this feature. 
the use cases I described already have some external system to guarantee the validity of event data against schemas and the compatibility of schema. 
I had thought about `AUTO_PRODUCE_BYTES(Schema<?> schema, boolean validate)` but decided that it would give this "dangerous option" to many formats that should never use it. 

I think it is better to add this feature in a case-by-case fashion.

Reply:
+1 with the current approach. I think we should make it more explicitly instead of introducing a flag to the `AUTO_PRODUC_BYTES` schema

Reply:
Agreed.

Thank you @sijie for your quick answer